internal/controller/drpolicy_controller.go

```diff
@@ -290,6 +284,11 @@ func validatePolicyConflicts(ctx context.Context,
 		return fmt.Errorf("validate managed cluster in drpolicy %v failed: %w", drpolicy.Name, err)
 	}
 
+	// DRPolicy does not support both Sync and Async configurations in one single DRPolicy
```

Comment:
(nit) I would move this check before line 282

Reply:
updated